addons/nuxeo-retention/nuxeo-retention-core/src/test/java/org/nuxeo/retention/test/TestRetentionManager.java

```diff
@@ -203,6 +203,18 @@ public void testManualPastMetadataBasedRule() throws InterruptedException {
         assertFalse(session.isUnderRetentionOrLegalHold(file.getRef()));
     }
 
+    @Test
+    public void testManualNullMetadataBasedRule() throws InterruptedException {
+        RetentionRule testRule = createManualMetadataBasedRuleMillis("dc:expired", 500L);
+        Calendar haldSecond = Calendar.getInstance();
+        haldSecond.add(Calendar.MILLISECOND, -1000);
```

Comment:
`createManualMetadataBasedRuleMillis("dc:expired", 500L);` -> `reateManualMetadataBasedRuleMillis("dc:expired", 500);` and `.... -1000);` -> ` -1_000);`